zscript/Monsters/ZombieMen/ZombieManCarbine.zc

```diff
@@ -830,8 +830,7 @@ Class PB_CarbineZombieman: PB_Zombieman //Replaces ZombieMan
 			TNT1 A 0 A_Jump (256, "DeathMinorhead1", "DeathMinorhead3", "DeathMinorhead4");
 		DeathMinorHead1:
 			TNT1 A 0 {
-				A_Scream();
-				A_NoBlocking();
+				A_ScreamAndUnblock();
```

Comment:
This anonymous function I recent found of the ZDoom Wiki combines both A_Scream and A_NoBlocking into one function, so I think this will be useful for certain death animations that currently use A_Scream and A_NoBlocking closer to each other (A_Scream first then followed by A_NoBlocking), so it looks more cleaner and less complex.